### DISCUSSION TITLE
CPU & RAM Stuck on 0

### DISCUSSION THREAD

**Discussion by vlsho** [pterodactyl/panel#5199] [Support] [answered]
=
![0](https://github.com/user-attachments/assets/0d7dc7f1-3fdd-45ff-bdc5-c79a70118ff3)

This problem with all my pterodactyl servers did anyone have any solution for that cause i didn't find i try everything but same thing 

i'm using debian 11

**Comment by rowansc1** [comment, marked as answer]
Please join the discord for support with this: https://discord.gg/pterodactyl
